lib/typescript/httpclient/README.md

```diff
@@ -4,19 +4,15 @@ The HttpClient Library includes a HTTP client for making requests to Airy's API.
 
 The library exports a HttpClient class with public methods that make requests to Airy's API.
 
-To use the library, you need to instantiate the class with the authentification token and your api url.
-
-The authentication token is optional, but all endpoints except for `/users.login` and `/users.signup` will fail without it.
+To use the library, you need to instantiate the class with an Airy Core API URL.
 
 For example:
 
-```
-import { HttpClient} from 'httpclient';
-
-const myInstance = new HttpClient(apiUrl, authToken);
-
-myInstance.listChannels()
+```typescript
+import {HttpClient} from "httpclient";
 
+const myInstance = new HttpClient("http://airy.core");
```

Comment:
since we're at it I'd rename this const to `client`? It reads better (super minor thing I get it:D)